content/modules/tour_guide/pages/meetings/meeting_notes.adoc

```diff
@@ -0,0 +1,50 @@
+= Meeting notes
+:description: Contains all information on meeting minutes
+:keywords: meeting, meeting_minutes
+:page-layout: progress // TODO
+:page-progress_bar: progress_bar_simple-polls_and_voting.svg // TODO
+:tour_guide_small:
+
+//tag::body[]
+
+include::tour_guide:partial$_attributes.adoc[]
+
+== Overview
+This page outlines ASAM's rules for meeting minutes and how to create them with issue templates.
+Meeting notes (minutes) are to be taken in the project's repository.
```

Comment:
I thought that was elegantly worded because it's so generic that it can be the project GitLab repository or the project SharePoint, which is also a repository. Or is that not how you can say it in English?

Reply:
The Guide distinguishes between the project repository (containing the source files of a the standard, GitLab or GitHub for Simulation) and Sharepoint.
Thus, the notation would be misleading here.

Reply:
Changed.